Mesh4all/m4a-firmware
# firmware/sys/shell_extended: Adds  support for `net_tools` commands

### Contribution description
Adds support for net_tools module into `shell-extended` module

### Testing procedure
Go to  `examples/shell_extended` and add the next line:
```
USEMODULE += net_tools
```
A good place for it could be just under `USEMODULE += uniqueid`, after it save changes, build and flash as usual:
```sh
make flash term
```
Once the shell is displayed, type `help` and push return or type one of the available commands for net-tools or `uniqueid`

### Issues/PRs references
None
